core/constants-dev.yaml

```diff
@@ -35,8 +35,14 @@ vssMaxTTL: 6 # epochs
 vssMinTTL: 2 # epochs
 mpcSendInterval: 10 # must be less than (2 * k * slotDuration - networkDiameter)
 
-# Lightweight delegation
+# Delegation
 lightDlgConfirmationTimeout: 500
+# This value parameterizes size of dlg cache. There are two types of messages:
```

Comment:
This comment doesn't explain anything. Size of this value -- in bytes? Multiples of default size of proxy cert/maximum of `(psk msg,confirmation msg)`?